pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -78,52 +81,85 @@ func (c *Cluster) makeDeployment(mgrConfig *mgrConfig, port int) *extensions.Dep
 	return d
 }
 
-func (c *Cluster) makeConfigInitContainer(mgrConfig *mgrConfig) v1.Container {
-	return v1.Container{
-		Name: opspec.ConfigInitContainerName,
+func (c *Cluster) makeCopyKeyringInitContainer(mgrConfig *mgrConfig) v1.Container {
+	// mgr does not obey `--keyring=/etc/ceph/keyring-store/keyring` flag, and always reports:
+	// "unable to find a keyring on /var/lib/ceph/mgr/ceph-a/keyring: (2) No such file or directory"
+	// Workaround: copy the keyring to the container data dir
+	container := v1.Container{
+		Name: "init-copy-keyring-workaround",
```

Comment:
How about dropping "workaround" from the container name? It might be a workaround, but no need to advertise it :)

Reply:
How does the mgr work today if it's not honoring the `--keyring` arg?

Reply:
To be honest, I don't know. When I mentioned the problem as I encountered it months ago, I was told that I shouldn't be seeing this behavior. So it could be a bug? @liewegas

Reply:
Update: It looks like the `--keyring` flag might only be ignored in Luminous. Mimic's logs seem to suggest that they look for the keyring in the flag path.

Reply:
Got it. Then we only need to create this init container for luminous?